sirepo/package_data/static/js/raydata.js

```diff
@@ -367,105 +375,71 @@ SIREPO.app.directive('scansTable', function() {
             modelName: '=',
         },
         template: `
-            <div data-show-loading-and-error="" data-model-key="scans">
+            <div class="row" data-show-loading-and-error="" data-model-key="scans">
               <div>
-                <button class="btn btn-info btn-xs" data-ng-click="addColumn()" style="float: right; margin:5px;"><span class="glyphicon glyphicon-plus"></span></button>
-                <button class="btn btn-info btn-xs" data-ng-show="showPdfButton()" data-ng-click="downloadSelectedAnalyses()" style="float: right; margin:5px;">Download Selected Analysis PDFs</button>
+                <div class="pull-right" data-ng-if="pageLocationText">
+                  <span class="raydata-button">{{ pageLocationText }}</span>
+                  <button class="btn btn-info btn-xs raydata-button" data-ng-click="pagePrevious()"><span class="glyphicon glyphicon-chevron-left"></span></button>
+                  <button class="btn btn-info btn-xs raydata-button" data-ng-click="pageNext()"><span class="glyphicon glyphicon-chevron-right"></span></button>
+                </div>
+                <button class="btn btn-info btn-xs raydata-button pull-right" data-ng-click="addColumn()"><span class="glyphicon glyphicon-plus"></span> Columns</button>
+                <button class="btn btn-info btn-xs raydata-button pull-right" data-ng-show="showPdfButton()" data-ng-click="downloadSelectedAnalyses()">Download Selected Analysis PDFs</button>
                 <table class="table table-striped table-hover">
                   <thead>
                     <tr>
-                      <th style="width: 20px; height: 20px;" data-ng-show="showPdfColumn"><input type="checkbox" data-ng-checked="pdfSelectAllScans" data-ng-click="togglePdfSelectAll()"/> PDF</th>
-                      <th data-ng-if="analysisStatus === 'allStatuses'" style="width: 50px; height: 40px;"></th>
-                      <th style="width: 50px; height: 40px;"></th>
-                      <th data-ng-repeat="column in columnHeaders track by $index" data-ng-mouseover="hoverChange($index, true)" data-ng-mouseleave="hoverChange($index, false)" data-ng-click="sortCol(column)" style="width: 100px; height: 40px;">
+                      <th style="width: 20px; height: 40px; white-space: nowrap" data-ng-show="showPdfColumn"><input type="checkbox" data-ng-checked="pdfSelectAllScans" data-ng-click="togglePdfSelectAll()"/> <span style="vertical-align: top">PDF</span></th>
```

Comment:
Are the notebooks still producing PDFs or can we get rid of this column?

Reply:
That's a good question. I think they will continue to be produced. They will just be written out to a configured location with the rest of the analysis and we will have to retrieve them from there.

Reply:
Good question - the old chx notebook we use does use the scan.pdf column, but it looks like csx does not. @e-carlin do you know if the current notebooks use this feature?

Reply:
From my understanding the latest notebooks both write out pdf's. So scan.pdf should be True for them.